File: PS Multi Tools/Classes/BytesConverter.vb
Public Class BytesConverter

    Public Shared Function ToLittleEndian(Value As UInteger) As Byte()
        Dim buffer As Byte() = BitConverter.GetBytes(Value)

        If Not BitConverter.IsLittleEndian Then
            Array.Reverse(buffer)
        End If

        Return buffer
    End Function

    Public Shared Function ToLittleEndian(Value As ULong) As Byte()
        Dim buffer As Byte() = BitConverter.GetBytes(Value)

        If Not BitConverter.IsLittleEndian Then
            Array.Reverse(buffer)
        End If

        Return buffer
    End Function

End Class

File: PSMultiTools/Classes/BytesConverter.cs
using System;

namespace PSMultiTools.Classes
{
    public class BytesConverter
    {

        public static byte[] ToLittleEndian(uint Value)
        {
            byte[] buffer = BitConverter.GetBytes(Value);

            if (!BitConverter.IsLittleEndian)
            {
                Array.Reverse(buffer);
            }

            return buffer;
        }

        public static byte[] ToLittleEndian(ulong Value)
        {
            byte[] buffer = BitConverter.GetBytes(Value);

            if (!BitConverter.IsLittleEndian)
            {
                Array.Reverse(buffer);
            }

            return buffer;
        }

    }
}